python/openassetio/constants.py

```diff
@@ -48,6 +48,10 @@
 ## will determine the URL to use for writing data for new entities
 kWillManagePath = 8
 
+## If True, and supported by a @ref host, a thumbnail will be generated by the
```

Comment:
[fix] 
```suggestion
## If True, and supported by a @ref host, a thumbnail will be generated by the
```
Subtle NBSP (how did you manage that?), but does affect the rendered output.

Reply:
Fixed [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/116/commits/34e2fdf021f156a3c1c9328c61b4027d3b1378b1)